src/kotlinSessions/SessionTwo.kt

```diff
@@ -0,0 +1,64 @@
+package kotlinSessions
+
+fun main() {
+
+    //---------------------------------Loops----------------------------------
+
+    // a loop is a block which is designed to be of a repeated execution
+    // so instead of saying
+    // println(x + 1)
+    // println(x + 1)
+    // we would commonly say
+    println("Test while loop")
+    var i: Int = 0
+    while (i < 2) {
+        i = i + 1
+        println(i)
+    }
+
+    // do while loop is the same as while loop but the first condition is a mandatory condition (ie is executed even if the condition is not true)
+    println("Test do while loop")
+    i = 0
+    do {
+        i = i + 1
+        println(i)
+    } while (i > 2)
+
+    // we can have a list of any data type
+    // in this example we have a list of names
+    val array = listOf("Ahmed", "Pavly", "Nabil")
```

Comment:
We've not take arrays yet

Reply:
Yep, but this is mainly to illustrate the usage of for each and iterating over items.

Reply:
If we are going to remove it, we have to remove for each too.

Reply:
Nb : Ranges are also types of incremental number format arrays.

Reply:
yes we will explain for each when we take arrays inshallah